portfolio/src/App.tsx

```diff
@@ -1,21 +0,0 @@
-import React, {Component} from 'react';
-import logo from './logo.svg';
-import './App.css';
-import Cover from './components/Cover';
-import Info from './components/Info';
-import Container from 'react-bootstrap/Container';
-import Projects from './components/Projects';
-
-class App extends Component {
-  render() {
-    return (
-      <Container fluid={true} className="App">
-        <Cover/>
-        <Info/>
-        <Projects/>
-      </Container>
-    );
-  }
-}
```

Comment:
Wherever you can, I would suggest learning functional components. They're all the rage now. (also im surprised create-react-app didnt initialize this as a functional component, I think you might be on an older version of CRA?)

```tsx
function App() {
    return <insert jsx here/>
}
```